mmseg/datasets/pipelines/transforms.py

```diff
@@ -240,6 +246,20 @@ def _random_scale(self, results):
     def _resize_img(self, results):
         """Resize images with ``results['scale']``."""
         if self.keep_ratio:
+            if self.min_size is not None:
+                # TODO: support H not equal to w in crop size
```

Comment:
# TODO: Now 'min_size' is an 'int' which means the minimum shape of images is (min_size, min_size,3).
It will be supported that the minimum size is that the width and height are not equal.